docs/source/advanced/plugin-configuration.mdx

```diff
@@ -205,6 +205,35 @@ by default).
 
 With the [Android Studio plugin](../testing/android-studio-plugin), you can also go to <kbd>Tools</kbd> | <kbd>Apollo</kbd> | <kbd>Download schema</kbd> which acts as a shortcut to these tasks.
 
+## Generated methods
+By default all Kotlin models, operations, fragments, and input objects are generated as data classes. This means that the compiler will
```

Comment:
"compiler" in these docs often refer to the GraphQL compiler, explicit that it's the Kotlin compiler here:

```suggestion
By default all Kotlin models, operations, fragments, and input objects are generated as data classes. This means that the Kotlin compiler will
```